lib/order.rb

```diff
@@ -0,0 +1,25 @@
+require './lib/takeaway.rb'
```

Comment:
Try using 'require_relative' for linking to other classes. It probably doesn't make a huge difference here, but I've had some very annoying bugs that took hours to solve - only to realise that 'require_relative' is supposed to be used for linking to other classes, and 'require' should be used for gems